### PR TITLE
Bump version and update changelog

### PR DESCRIPTION
**Pull request by svix-lucho**
## Version 1.16.0
* Server: Add 'tag' parameter to list-message for go,js,python.
* Server: backport expired message cleaner updates.
* Libs/Javascript: Fix thrown error message when API status code is unknown.
* Server: expired message cleaner cleanup.
* Libs: fix descriptions in package repositories.
* Add download badges to README.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 ## Next
 * 
 
+## Version 1.16.0
+* Add download badges to README.
```
Comment (tasn) [inline ChangeLog.md:7]:
```suggestion
```
Our customers don't care about this. The Changelog should be a summary that only includes the things our customers care about.

File: ChangeLog.md [hunk 2]
```diff
@@ -3,6 +3,15 @@
...
+* Server: Add 'tag' parameter to list-message for go,js,python.
```
Comment (tasn) [inline ChangeLog.md:8]:
```suggestion
* Server: Add `tag` parameter to list-message for Go, JavaScript, and Python.
```
This is customer facing so should be nicely formatted.

File: ChangeLog.md [hunk 3]
```diff
@@ -3,6 +3,15 @@
...
+* Server: backport expired message cleaner updates.
+* Server: expired message cleaner cleanup.
+* Libs: fix descriptions in package repositories.
```
Comment (tasn) [inline ChangeLog.md:11]:
```suggestion
```

**Comment by svix-lucho** [comment]
Don't merge yet, I need to add a fix in the JS library

**Review by svix-aaron1011 — APPROVED** [review]

**Pull request merged**